elements/nuxeo-app/nuxeo-expired-session.js

```diff
@@ -36,11 +36,10 @@ Polymer({
         height: var(--nuxeo-drawer-header-height);
         padding-left: 16px;
         position: fixed;
-        top: 0;
+        top: var(--nuxeo-app-top);
```

Comment:
I think we should add the default value here, to make it more robust
```suggestion
        top: var(--nuxeo-app-top, 0px);
```

Reply:
agreed! will add a default value for the var occurrences